keymanapp/keyman
# chore(android,oem/fv/android): Update targetSDKVersion to 30 🍒

Fixes #5932. 🍒 -pick of #5126 and parts of #5127 to stable-14.0

To comply with Play Store requirement, this updates targetSDKVersion to 30

* Also sets appropriate webView permissions to access local file (except for WebBrowserActivity, the in-app browser).

## User Testing
For all the following, use Android emulator (e.g. Pixel 2) with Android API 30

* **TEST_KEYMAN** - Test Keyman app and system keyboard
1. Load Keyman
2. In "Get Started" menu, enable Keyman as a system keyboard and set as default keyboard
3. Exit the menu
4. Verify in-app keyboard appears
5. From Settings, open "Info"
6. Verify Keyman for Android help appears
7. Back out to the text area
8. From the menu, launch the in-app browser
9. Click on a text area and verify the Keyman system keyboard appears
10. Back out to the text area
11. From the "Settings" menu, install another keyboard from keyman.com
12. Proceed through the keyboard installation and verify keyboard installs

* **TEST_KMSample1** - Test KMSample1 app
1. Load KMSample1
2. Verify the in-app keyboard appears
3. Type and verify text is output

* **TEST_KMSample2** - Test KMSample2 app
1. Load KMSample2
2. In the menu, set KMSample2 as a system keyboard and set as default keyboard
3. Open a separate app (e.g. Chrome)
4. Click on a text area and verify the KMSample2 system keyboard appears

* **TEST_FV** - Test FirstVoices Android app
1. Load the FV app
2. Proceed through the Setup steps of selecting keyboard(s) and setting FirstVoices as the default system keyboard
3. Open a separate app (e.g. Chrome)
4. Click on a text area and verify the FirstVoices system keyboard appears
